src/comic-issue/dto/wallet-comic-issue.dto.ts

```diff
@@ -1,4 +1,4 @@
-import { IsBoolean, IsDate, IsPositive } from 'class-validator';
+import { IsBoolean, IsDate, IsOptional, IsPositive } from 'class-validator';
```

Comment:
It was working without this change (didn't wanted to make a change inside WalletComicIssue dto, my guess is that plainToClass is actually not going deeply in object keys)